src/store/ChartTypeStore.js

```diff
@@ -173,7 +174,9 @@ export default class ChartTypeStore {
             this.stx.chart.tension = 0;
             delete this.stx.layout.tension;
             if (this.aggregates[type.id]) {
+                this.stx.chart.baseline.userLevel = false;
                 this.stx.setAggregationType(type.id);
+                this.stx.chart.baseline.userLevel = true;
```

Comment:
i dont know when `basseline` need to be adjust so i just set it back to its default value after chart type changes.